client/app/components/group/_pickupList/pickupList.controller.js

```diff
@@ -59,6 +57,15 @@ class PickupListController {
     this.filterAndDisplayPickups();
   }
 
+/**
+ * checks if a pickup is already full
```

Comment:
nice documentation! We didn't do it so far, but maybe we should. Do you know how this style is called?